packages/chart/src/components/Legend/Legend.Suppression.tsx

```diff
@@ -95,10 +95,31 @@ const LegendSuppression: React.FC<LegendProps> = ({ config, isLegendBottom }) =>
   const getLegendContainerClass = () =>
     legend.singleRow && isLegendBottom ? 'legend-container__inner bottom single-row' : ''
 
+  const sumSeries = (data, seriesKey) => {
+    if (!Array.isArray(data)) return 0
+
+    return data.reduce((total, row) => {
+      const raw = row[seriesKey]
+      const num = parseFloat(raw)
+      return total + (Number.isFinite(num) ? num : 0)
+    }, 0)
+  }
+  const total = sumSeries(data, config.runtime.yAxis.dataKey)
+
   const shouldShowSuppressedInfo = () =>
     !config.legend.hideSuppressionLink &&
     config.visualizationSubType !== 'stacked' &&
     preliminaryData?.some(pd => pd.label && pd.type === 'suppression' && pd.value && (pd?.style || pd.symbol))
+  // controls Pie chart Legend  for calculated Area
+
+  const renderCalculatedAreaItems = () => {
+    return (
+      <div key={'iii'} className='legend-preliminary'>
```

Comment:
Can you change the key here to pie-asterisk or something thats lets ambiguous for future developers?